build/scripts/build.ps1

```diff
@@ -513,9 +514,12 @@ function Run-SignTool() {
     try {
         $signTool = Join-Path (Get-PackageDir "RoslynTools.Microsoft.SignTool") "tools\SignTool.exe"
         $signToolArgs = "-msbuildPath `"$msbuild`""
-        if (-not $official) {
-            $signToolArgs += " -test"
+        switch ($signType) {
+            "real" { break; }
+            "test" { $signToolArgs += " -testSign"; break; }
+            default { $signToolArgs += " -test"; break; }
```

Comment:
I must admit this switch doesn't make sense to me. I get the test to testSign line, but default just throws in -test? Is that a typo?

Reply:
(I suspect my real statement here is "please add comments")

Reply:
There are two different features here:

- "-test": this is similar to "-n" in many unix commands. It runs the SignTool in verification mode. No signing is done here. Instead it simply verifies the consistentcy of the build output with the information in the configuration file. 
- "-testSign": the signing mechanism is test signing, not real signing.

In retrospect I should have called "-test" "-verifyOnly". May still change it but worry that will cause a lot of churn.